docs/guide/overview.asciidoc

```diff
@@ -38,6 +36,42 @@ This data is buffered for a short period and sent on to APM Server.
 It listens on port 8200 by default and receives data from agents through a JSON HTTP API.
 Then it creates documents from that data and stores them in Elasticsearch.
 
-To visualize the data after it's sent to Elasticsearch,
-you can use the the dedicated APM UI bundled in X-Pack,
-or the pre-built open source Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
\ No newline at end of file
+*Elasticsearch* is a highly scalable open-source full-text search and analytics engine.
+It allows you to store, search, and analyze big volumes of data quickly and in near real time.
+
+*Kibana* is an open source analytics and visualization platform designed to work with Elasticsearch.
+You use Kibana to search, view, and interact with data stored in Elasticsearch indices.
+You an use Kibana to visualize APM data by utilizing the dedicated APM UI bundled in Basic license,
+or the pre-built, open source,
+Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
+
+[[concepts]]
+[float]
+=== APM Concepts
+
+*Agents*
+APM Agents capture different types of information from within their instrumented applications, known as events. Events can be <<transactions,`Transactions`>>, <<transaction-spans,`Spans`>>, and <<errors,`Errors`>>.
+These events are then streamed to a single endpoint in the APM Server which sorts and processes the events. 
+
+* <<transactions,`Transactions`>> - A transaction describes an event captured by an Elastic APM agent instrumenting a service. For example: an HTTP request, or a background job.
```

Comment:
Yeah, though traces are not events them selfs, but exist only indirectly because of the collected spans/transactions. So we can't list them in the list of events above

Reply:
We could add something like:
```
*Traces*, while not an event, play an important role in grouping events together. A Trace is..
```

Reply:
Added in DT section